### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ismangil**

### CODE REVIEW

**Comment by trengginas** [comment]
perhaps we need these files too? (e.g: CODE_OF_CONDUCT.md + CONTRIBUTING.md) https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file

**Comment by sauwming** [comment]
So is Contributor Agreement still needed to be signed? Or now reading it only is sufficient?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (nanangizz) [inline .github/ISSUE_TEMPLATE/bug_report.md]: More 'realistic' sample perhaps, also preferably using pjsua app, e.g:
1. run pjsua app with parameter: `--video --registrar sip:some.sip.registrar ...`
1. make two video calls to: `sip:abc` and `sip:xyz`1
1. setup video conference: `vid conf cc ...`
1. assertion occurs!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- OS, Distribution & Version: [e.g. MacOS Mojave, Ubuntu 16.04, iOS 13]
+- Logs
+- PJSIP Version [e.g. 2.10]
+
+**Smartphone (please also complete the following information):**
+
+- Device: [e.g. iPhone6]
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (nanangizz) [inline .github/ISSUE_TEMPLATE/bug_report.md]: titled 'Environment setup' perhaps?
 - OS version & dist: ...
 - Device/specific hardware info: ...
 - PJSIP
    - version: ...
    - applied patch(es): [e.g: patches from issue/PR xyz]
    - `configure` script params: ...
    - `config_site.h` contents: ...
    - related third party libraries & versions: [e.g: OpenSSL 1.1.1b]
 - Additional context: ...

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (nanangizz) [inline .github/ISSUE_TEMPLATE/bug_report.md]: PJSIP log with verbosity level 5, packet capture (e.g: using Wireshark), audio/video recording, screenshot, etc

**Comment by ismangil** [comment]
@sauwming CLA assistant will ask and say that they are digitally agreeing to the agreement.

**Pull request merged**